android/app/proguard-rules.pro

```diff
@@ -15,3 +15,64 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
+# Disabling obfuscation is useful if you collect stack traces from production crashes
+# (unless you are using a system that supports de-obfuscate the stack traces).
+#-dontobfuscate
+
+# React Native
```

Comment:
if you copied the following rules from some reference, please add the link in the comment

Reply:
up to line 64 is the official [Proguard.pro](https://github.com/facebook/react-native/blob/master/ReactAndroid/proguard-rules.pro) present in the Facebook repository, it must be added?

Reply:
yes please provide a link to the source, it could be useful for future reference, thanks